x-pack/auditbeat/module/system/socket/socket.go

```diff
@@ -272,17 +303,46 @@ func (ms *MetricSet) reportChanges(report mb.ReporterV2) error {
 	return nil
 }
 
-func socketEvent(socket *Socket, eventType string, eventAction string) mb.Event {
+func socketEvent(socket *Socket, eventType string, action eventAction) mb.Event {
 	event := mb.Event{
 		RootFields: socket.toMapStr(),
 	}
 
 	event.RootFields.Put("event.type", eventType)
-	event.RootFields.Put("event.action", eventAction)
+	event.RootFields.Put("event.action", action.String())
+	event.RootFields.Put("message", socketMessage(socket, action))
 
 	return event
 }
 
+func socketMessage(socket *Socket, action eventAction) string {
+	var actionString string
+	switch action {
+	case eventActionSocketOpened:
+		actionString = "OPENED"
+	case eventActionSocketClosed:
+		actionString = "CLOSED"
+	case eventActionExistingSocket:
+		actionString = "OPEN"
```

Comment:
Or LISTENING?

Reply:
Well, it will already say `Listening socket (:::22) OPEN by process sshd (PID: 966) and user root (UID: 0)`.

Do we want it to say `Listening socket (:::22) LISTENING by process sshd (PID: 966) and user root (UID: 0)`, i.e. 2x "listening"?

Reply:
Ah, in the context of the full string, I agree this is perfectly fine 👍